package.json

```
@@ -76,15 +77,15 @@
     "qunit-dom": "^2.0.0",
     "release-it": "^15.4.0",
     "release-it-lerna-changelog": "^5.0.0",
-    "sass": "^1.54.5",
+    "sass": "^1.69.7",
     "stylelint": "^15.4.0",
     "stylelint-config-standard": "^34.0.0",
     "stylelint-prettier": "^3.0.0",
     "webpack": "^5.78.0",
     "xml-formatter": "^2.6.1"
   },
   "peerDependencies": {
-    "@appuniversum/ember-appuniversum": "^2.0.0",
+    "@appuniversum/ember-appuniversum": "^2.0.0 || ^3.0.0",
```

does this not need to be ?

```suggestion
    "@appuniversum/ember-appuniversum": "^2.17.0 || ^3.0.0",
```

---

This does not really seem necessary to me? As the style import only happens in the dummy/consuming app, not in the addon itself.

---

`^2.17.0` is needed for the dummy app due to how it imports the sass from appuniversum, but there's no change to the older version support of the addon itself.